bindings/python/example_scripts/account_analysis.py

```diff
@@ -126,7 +126,7 @@ def period_end(start_year, start_month, period_type):
     # so we get a period end like
     # 2010-03-31 for period starting 2010-01 instead of 2010-04-01
     return date(end_year, end_month, 1) - ONE_DAY
-    
+
 
 def generate_period_boundaries(start_year, start_month, period_type, periods):
     for i in xrange(periods):
```

Comment:
Shouldn't xrange be changed to range?

Reply:
Good point. I'm sure there are some more things like this.